src/main/resources/data/absentbydesign/recipes/stairs_smooth_stone_sc.json

```diff
@@ -4,6 +4,6 @@
   "ingredient": {
     "item": "minecraft:smooth_stone"
   },
-  "result": "absentbydesign:smooth_stone",
```

Comment:
I think result should be something like 

"result" "absentbydesign:stairs_smooth_stone",